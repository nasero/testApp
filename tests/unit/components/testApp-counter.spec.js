import { shallowMount } from '@vue/test-utils';
import TestAppCounter from '@/components/testApp-counter';

describe('TestAppCounter - Snapshot', () => {
    it('Snapshot', () => {
        const wrapper = shallowMount(TestAppCounter);
        expect(wrapper.element).toMatchSnapshot();
    });
});
