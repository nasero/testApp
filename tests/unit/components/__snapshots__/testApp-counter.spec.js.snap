// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TestAppCounter - Snapshot Snapshot 1`] = `
<div
  class="testApp-counter"
>
  <p
    class="testApp-counter__digit"
  >
    
  </p>
  <p
    class="testApp-counter__label"
  >
    
  </p>
</div>
`;
